docker/istio_builders/Makefile

```diff
@@ -1,5 +1,5 @@
 PROJECT = istio-testing
-VERSION = 0.5.11
+VERSION = 0.5.12
```

Comment:
revert this changes as this directory is not used anymore.

Reply:
done